.github/workflows/build-ci.yml

```diff
@@ -0,0 +1,24 @@
+name: PMIx Standard CI
+
+on:
+  pull_request:
+    # We don't need this to be run on all types of PR behavior
+    # See
+    #  - https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request
+    #  - https://frontside.com/blog/2020-05-26-github-actions-pull_request
+    types:
+      - opened
+      - synchronize
+      - edited
```

Comment:
What's the motivation for re-running on `edited`?  Based on the second link in the comment, `edited` means "title, body, or the base branch of the PR is modified".

W.r.t the defaults, this is missing `reopened`, is that intentional?

Reply:
I included `edited` because of the base branch modification. So if they switch the PR from targeting `v4` to `master` then we should probably rerun it.

`reopened` is a good idea. I'll add that.

Reply:
Done in the latest push.